mungegithub/github/github.go

```diff
@@ -1574,6 +1574,21 @@ func (obj *MungeObject) GetPR() (*github.PullRequest, bool) {
 	return pr, true
 }
 
+// Returns true if the user is in the list of removed assignees
```

Comment:
this comment is to specific.  the word `removed` has nothing to do with what this func does